src/pykeen/models/base.py

```diff
@@ -391,6 +391,9 @@ def predict_r(
         """
         self.eval()  # Enforce evaluation mode
         ht_batch = ht_batch.to(self.device)
+        # TODO is this applicable?
+        # if self.use_inverse_triples:
```

Comment:
not really. We have `score_h_inverse`, since we have inverse *triples*, where the order of *entities* is inverted, and the *relation* replaced by an inverse *relation*.